ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -20,12 +20,43 @@
 from abc import ABC, abstractmethod
 from typing import List, NamedTuple, Tuple, Optional, Union, Dict
 import numpy as np
-from enum import Enum
+from enum import Enum, IntEnum
 
 AgentId = int
 AgentGroup = str
 
 
+class EpisodeStatus(IntEnum):
+    """
+    The Episode is still ongoing
+    """
+
+    Default = 0
```

Comment:
Python enum names should be ALL_UPPER_CASE (source: https://mail.python.org/pipermail/python-ideas/2016-September/042340.html and https://stackoverflow.com/a/21359677/224264)